webknossos/webknossos/geometry/vec3_int.py

```diff
@@ -44,6 +62,11 @@ def from_xyz(x: int, y: int, z: int) -> "Vec3Int":
         # the tolerant (and potentially) slow Vec3Int.__new__ method.
         return tuple.__new__(Vec3Int, (x, y, z))
 
+    @staticmethod
+    def from_vec3_float(vec: Tuple[float, float, float]) -> "Vec3Int":
+
+        return Vec3Int(floor(vec[0]), floor(vec[1]), floor(vec[2]))
```

Comment:
I'm not quite sure why this should be floored, I'd either expect `int()` (which only differs on negative inputs from floor, which probably won't be the case), or `round()`.

Reply:
You are right. I changed it to `int'.